### PR TITLE
Image upload

### PR DESCRIPTION
**Pull request by hwyann**
Cloudinary implemented and image upload function added to the Match create form

### CODE REVIEW

File: config/storage.yml [hunk 1]
```diff
@@ -1,2 +1,36 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
```
Comment (tomonorman) [inline config/storage.yml]: I think the above needs to be deleted  as well

**Review by tomonorman — CHANGES_REQUESTED** [review]
Delete some shit

File: config/storage.yml (hunk 1 above)
Comment (hwyann) [reply config/storage.yml]: deleted!

**Review by tomonorman — APPROVED** [review]
good good

**Pull request merged**